src/utils/py/daos_api.py

```diff
@@ -742,7 +742,7 @@ def create(self, rank=None, objcls=13):
         func = self.context.get_function('generate-oid')
 
         # XXX convert object class
-	cls = int(ConvertObjClass[DaosObjClassOld(objcls)])
+        cls = int(ConvertObjClass[DaosObjClassOld(objcls)])
```

Comment:
Should `objcls=13` on line 740 be `objcls=DaosObjClassOld.DAOS_OC_ECHO_R3S_RW` or one of the new DaosObjClass enumerations, instead of `13` as the enumeration value has changed?

Also I think we should support both str and int object class inputs:
`cls = int(ConvertObjClass[DaosObjClassOld[objcls] if isinistance(objcls, str) else DaosObjClassOld(objcls)])`

Reply:
@phender Totally agree with all your comments. They will be addressed in DAOS-2786. Thanks.

Reply:
Great, thanks.